### ISSUE TITLE
Nice work! Any information about the metrics at 30k? 

### ISSUE THREAD

**Issue by wanmeihuali** [WangFeng18/3d-gaussian-splatting#4]
Thank you for your great work! 
I also have an implementation using Taichi Lang: https://github.com/wanmeihuali/taichi_3d_gaussian_splatting
However, I think both our PSNR metric is a bit worse than what the paper claims. So I'm wondering what's their "trick"....
I have the following questions:
- Have you measured the PSNR/SSIM at 30k? 
- Also, what's your resolution used for the garden dataset? 
- what's the number of points at 7k/30k?
I did notice some slight differences between our implementation, e.g. you provide a weight_normalize option, I'm also wondering does enable weight_normalize help ?

**Comment by WangFeng18** [comment]
Hi, thanks for your attention. 
I have not yet test the 30k iteration, the adaptive control is still problematical (very sensible to many hyperparameters such as learning rate, resolution) and I am tring to fix some problems.
The resolution for garden is 4x downsample
I will update the README to give a concrete record of gaussian numbers, currently 500-1000k gaussian numbers are usual.
weight_normalize does not affect the performances.
I hope it can help. I will work in progress and update the README doc

**Comment by wanmeihuali** [comment]
Thanks for your information!
